### PR TITLE
feat(clickup): Get Task Action / Get Tasks Action / Task Tag Updated Trigger

### PR DESCRIPTION
**Pull request by kanarelo**
## What does this PR do?

On task tag update, there should be a trigger for the action.

adds TASK_TAG_UPDATED trigger

ANNOUNCEMENT=true

### CODE REVIEW

File: packages/pieces/clickup/src/lib/triggers/task-tag-updated.ts [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        type: AuthenticationType.BEARER_TOKEN,
+        token: context.auth.access_token
+      },
+      queryParams: {},
+    }
+
+    const response = await httpClient.sendRequest<WebhookInformation>(request);
+    console.debug(`clickup.${ClickupEventType.TASK_TAG_UPDATED}.onEnable`, response)
+
+    await context.store.put<WebhookInformation>(`clickup_${name}_trigger`, response.body);
```
Comment (kishanprmr) [inline packages/pieces/clickup/src/lib/triggers/task-tag-updated.ts:72]:
```suggestion
    await context.store.put<WebhookInformation>(`clickup_task_tag_updated_trigger`, response.body);
```
name variable is not defined,you can use suggested one.

File: packages/pieces/clickup/src/lib/triggers/task-tag-updated.ts [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+          username: 'Activepieces Apps',
+          email: 'apps@activepieces.com',
+          color: '#aa2fff',
+          initials: 'AA',
+          profilePicture: null,
+        },
+        before: null,
+        after: '90040005586783',
+      },
+    ],
```
Comment (kishanprmr) [inline packages/pieces/clickup/src/lib/triggers/task-tag-updated.ts]: can you please change sampledata for Tag changed webhook? you can use [this](https://clickup.com/api/developer-portal/webhooktaskpayloads/#tasktagupdated-payload) from clickup documnetation.

File: packages/pieces/clickup/src/lib/triggers/task-tag-updated.ts (hunk 1 above)
Comment (kishanprmr) [inline packages/pieces/clickup/src/lib/triggers/task-tag-updated.ts]: name variable is not defined.you can use suggested one.
```suggestion
    await context.store.put<WebhookInformation>(`clickup_task_tag_updated_trigger`, response.body);
```

File: packages/pieces/clickup/src/lib/triggers/task-tag-updated.ts [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+      queryParams: {},
+    }
+
+    const response = await httpClient.sendRequest<WebhookInformation>(request);
+    console.debug(`clickup.${ClickupEventType.TASK_TAG_UPDATED}.onEnable`, response)
+
+    await context.store.put<WebhookInformation>(`clickup_${name}_trigger`, response.body);
+  },
+  async onDisable(context) {
+    const webhook = await context.store.get<WebhookInformation>(`clickup_${name}_trigger`);
```
Comment (kishanprmr) [inline packages/pieces/clickup/src/lib/triggers/task-tag-updated.ts]:
```suggestion
    const webhook = await context.store.get<WebhookInformation>(`clickup_task_tag_updated_trigger`);
```

Comment (kanarelo) [reply packages/pieces/clickup/src/lib/triggers/task-tag-updated.ts]: Thanks

File: packages/pieces/clickup/src/lib/triggers/task-tag-updated.ts (hunk 2 above)
Comment (kanarelo) [reply packages/pieces/clickup/src/lib/triggers/task-tag-updated.ts]: Thanks, fixed, sorry I am a bit rusted.

**Pull request merged**